includes/ppc-frontend.php

```diff
@@ -5,16 +5,18 @@
  *
```

Comment:
Prevent direct access to single PHP files as this one. [https://www.barrykooij.com/prevent-direct-file-access-plugin-files/](url)

Reply:
added 
if ( ! defined( 'ABSPATH' ) ) {
	exit; // Exit if accessed directly.
}
at the start of ppc-frontend.php file.